src/methods/clipping/clipping_processor.jl

```diff
@@ -24,16 +44,20 @@ returns are the fully updated vectors of PolyNodes that represent the rings 'pol
 'poly_b', respectively. This function also returns 'a_idx_list', which at its "ith" index
 stores the index in 'a_list' at which the "ith" intersection point lies.
 =#
-function _build_ab_list(::Type{T}, poly_a, poly_b) where T
+function _build_ab_list(::Type{T}, poly_a, poly_b, delay_cross_f, delay_bounce_f) where T
```

Comment:
```suggestion
function _build_ab_list(::Type{T}, poly_a, poly_b, delay_cross_f::F1, delay_bounce_f::F2) where {T, F1, F2}
```
in combination with the previous suggestion fixes the problem.  I guess the compiler needed a hint to specialize.

Reply:
```julia
# Original code
julia> @benchmark GO.intersection($p25, $p26; target = $GI.PolygonTrait())
BenchmarkTools.Trial: 10000 samples with 1 evaluation.
 Range (min … max):   9.625 μs …  4.559 ms  ┊ GC (min … max): 0.00% … 99.05%
 Time  (median):     10.000 μs              ┊ GC (median):    0.00%
 Time  (mean ± σ):   11.276 μs ± 64.090 μs  ┊ GC (mean ± σ):  7.84% ±  1.40%

  ▃▆▇█▇▅▄▂▁▁  ▁▁▁▁                                            ▂
  ██████████████████▇▆▆▆▆▆▅▆▄▅▆▁▆▄▆▆▆▅▆▅▇▅▅▅▃▅▆▃▆▃▃▄▄▅▄▅▅▅▃▅▄ █
  9.62 μs      Histogram: log(frequency) by time      16.1 μs <

# With the previous suggestion
julia> @benchmark GO.intersection($p25, $p26; target = $GI.PolygonTrait()) evals=15
BenchmarkTools.Trial: 10000 samples with 15 evaluations.
 Range (min … max):  10.200 μs … 334.086 μs  ┊ GC (min … max): 0.00% … 95.99%
 Time  (median):     10.464 μs               ┊ GC (median):    0.00%
 Time  (mean ± σ):   11.135 μs ±   9.148 μs  ┊ GC (mean ± σ):  4.59% ±  5.39%

    ▄█▄                                                         
  ▂▅████▇▇▇▅▄▄▄▃▃▂▂▂▂▂▂▂▂▂▂▂▁▂▂▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁ ▂
  10.2 μs         Histogram: frequency by time         12.7 μs <

# Both suggestions
julia> @benchmark GO.intersection($p25, $p26; target = $GI.PolygonTrait()) evals=15
BenchmarkTools.Trial: 10000 samples with 15 evaluations.
 Range (min … max):  8.933 μs … 323.664 μs  ┊ GC (min … max): 0.00% … 95.86%
 Time  (median):     9.147 μs               ┊ GC (median):    0.00%
 Time  (mean ± σ):   9.790 μs ±   8.926 μs  ┊ GC (mean ± σ):  4.90% ±  5.35%

     ▅█▇                                                       
  ▁▂▅████▅▅▅▅▄▃▂▃▃▂▂▂▂▂▂▂▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁ ▂
  8.93 μs         Histogram: frequency by time          11 μs <

 Memory estimate: 16.36 KiB, allocs estimate: 110.